examples/load-reporting-service/docker-compose.yaml

```
@@ -14,7 +14,7 @@ services:
       context: .
       dockerfile: Dockerfile-envoy
     ports:
-    - "${PORT_PROXY0:-80}-${PORT_PROXY0:-81}:80"
+    - "${PORT_PROXY0:-80}-${PORT_PROXY1:-81}:80"
```

```suggestion
    - "${PORT_PROXY0:-80}-${PORT_PROXY1:-81}:80"
```

this change seems correct to me - altho im wondering if/how it is currently working

---

```shell
Step 1: Build the sandbox
Terminal 1

$ pwd
envoy/examples/load-reporting-service
$ docker-compose pull
$ docker-compose up --scale http_service=2

---

Step 2: Start sending stream of HTTP requests
Terminal 2

$ pwd
envoy/examples/load_reporting_service
$ bash send_requests.sh
```

I executed the send_requests.sh script in the example and got some errors.

---

i see - its only working in CI, not in reality